pkgs/tools/compression/gzip/default.nix

```diff
@@ -30,10 +30,10 @@ stdenv.mkDerivation rec {
     }' $out/bin/*
   '';
 
-  # set GZIP env variable to "-n" to stop gzip from adding timestamps
+  # use flag "-n" to stop gzip from adding timestamps
   # to archive headers: https://github.com/NixOS/nixpkgs/issues/86348
   setupHook = writeText "setup-hook" ''
-    export GZIP="-n"
+    alias gzip="gzip -n"
```

Comment:
are aliases exported to subprocesses like GZIP is?

Reply:
That's a good point, I don't think so.